aspnet/Treegrid/Context-Menu.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Context Menu | TreeGrid | ASP.NET | Syncfusion
-description: context menu
+description: Learn here all about Context menu in Syncfusion Essential ASP.NET Web Forms TreeGrid control, its elements and more details.
```

Comment:
Learn here all about context menu support in Syncfusion Essential ASP.NET Web Forms TreeGrid control, its elements, and more details.

Reply:
Updated